src/dfind/dfind.c

```diff
@@ -338,7 +356,7 @@ int main (int argc, char** argv)
     int usage = 0;
     while (1) {
         int c = getopt_long(
-                    argc, argv, "i:o:vqh",
+                    argc, argv, "i:o:tvqh",
```

Comment:
@daltonbohning The lowercase 't' option is added here but not 'T'?

Reply:
@dsikich Right. That's because the uppercase 'T' is for --type. But the command line only takes the long option for --type, like with these:
https://github.com/hpc/mpifileutils/blob/2fc898231298484692385e12512ba8189c0c6cc8/src/dfind/dfind.c#L328-L347

The 'T' is only used internally for parsing the arguments.
Basically, if the print_usage doesn't show the short options, then the short options don't appear in this list because they aren't exposed through the command line.

Reply:
@daltonbohning oh ok it only takes the long option I missed that. thanks! LGTM.

Reply:
No worries :)